sections/02_intro.md

```diff
@@ -34,88 +34,118 @@ learning methods more challenging or less fruitful to apply.
 ### What is deep learning?
 
 One version:
-[This section needs citations]
+`TODO: this section needs citations`
 
-Deep Learning is a collection of new techniques that together have recently demonstrated 
-breakthrough gains over existing approaches in several fields. 
+Deep Learning is a collection of new techniques that together have recently
+demonstrated  breakthrough gains over existing approaches in several fields.
 
 Deep learning is built on a very old idea, neural networks, that was first
```

Comment:
Suggest merging these two paragraphs for "sound", e.g. Deep learning is a collection ... It is built upon ...